src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -10,8 +10,10 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic.SyntaxFacts
 Imports Roslyn.Test.Utilities
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module ParserTestUtilities
+    Friend ReadOnly Property PooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool()
```

Comment:
I'm using the timings reports from test runs, on jenkins.
Focusing on the unittests
```
* Microsoft.CodeAnalysis.VisualBasic.UnitTests
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.AnonymousDelegates	
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Binding
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.CodeGen
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Editting
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Emit
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.ExtensionMethods
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.FlowAnalysis
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Formatting
* Microsoft.CodeAnalysis.VisualBasic.UnitTests.Microsoft.CodeAnalysis.VisualBasic.UnitTests	
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.PDB
* Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Symbols
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Symbols.CorLibrary	
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Symbols.Metadata	
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Symbols.Metadata.PE
  Microsoft.CodeAnalysis.VisualBasic.UnitTests.Symbols.Retargeting
```
Especially those that are starred, as these take the longest.